lib/commands/stats.js

```diff
@@ -0,0 +1,54 @@
+const { humanizeTime, getCPULoad } = require("../core/Utils");
+
+module.exports = {
+    permissions: ["developer"],
+    hidden: true,
+    guildOnly: false,
+    cooldown: 5000,
+    shortDescription: 'Outputs bot statistics',
+    longDescription: 'Outputs relevant bot-related statistics.\n\nOnly developers have access to this command.',
+    run: async function(client, m, args) {
+        let repoURL = require("../../package.json").repository.url.slice(4, -4);
+        let { version } = require("../../package.json");
+        let latestCommit = require('child_process').execSync('git rev-parse HEAD').toString().trim();
```

Comment:
From [the nodejs docs](https://nodejs.org/api/child_process.html#child_process_synchronous_process_creation): `The child_process.spawnSync(), child_process.execSync(), and child_process.execFileSync() methods are synchronous and WILL block the Node.js event loop, pausing execution of any additional code until the spawned process exits.`

If you can make this async instead, it would be great.